src/polar/Pie.tsx

```diff
@@ -80,7 +80,7 @@ export type PieSectorData = {
   value?: number;
   paddingAngle?: number;
   dataKey?: string;
-  payload?: any[];
```

Comment:
?

Reply:
I think this type was wrong from the start. In the storybook it receives an entry object, not array.